src/strategies/decentraland-wearable-rarity/README.md

```diff
@@ -0,0 +1,47 @@
+# Decentraland Wearable Rarity
+
+This strategy allows users to calculate the total amount of Decentraland wearables and apply a multiply to each rarity. Additionally you can limit the collections that will be take into account using a list of ids.
```

Comment:
Typo, change "multiply" to "multiplier".
Also "that will be taken into account".